domain/src/test/kotlin/no/nav/su/se/bakover/domain/oppdrag/utbetaling/UtbetalingsstrategiNyTest.kt

```diff
@@ -359,6 +432,556 @@ internal class UtbetalingsstrategiNyTest {
         )
     }
 
+    @Test
+    fun `kaster exception hvis månedsbereging og uføregrunnlag er tomme (0 til 0)`() {
```

Comment:
```suggestion
    fun `kaster exception hvis månedsberegning og uføregrunnlag er tomme (0 til 0)`() {
```
Gjelder flere nedover her